plugins/modules/docker_image.py

```diff
@@ -549,9 +557,26 @@ def archive_image(self, name, tag):
             self.log("archive image: image %s not found" % image_name)
             return
 
+        # Will have a 'sha256:' prefix
+        new_image_id = image['Id']
+
+        try:
+            archived_id = archived_image_id(self.archive_path)
+
+            if archived_id:
+                log.debug('Previous image ID was %s', archived_id)
```

Comment:
Please use `self.log()` or - if really necessary - `module.debug()`.

Reply:
Changed to self.log, but would like to know how to call `module.debug` to report the exception.